app/controllers/admin/companies_controller.rb

```diff
@@ -7,4 +7,11 @@ def destroy
     Company.find(params[:id]).destroy
     redirect_to admin_dashboard_index_path
   end
+
+  def update
+    company = Company.find(params[:id])
+    company.update(status: params[:company][:status])
+    company.save
```

Comment:
I am fine with it as long as it's hard coded and has no chance of not saving. This is just the admin changing the status of a company via dropdown right?